test/nut/commands/force/org/org.nut.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import * as querystring from 'querystring';
+import { expect } from '@salesforce/command/lib/test';
+import { TestSession } from '@salesforce/cli-plugins-testkit';
+import { execCmd } from '@salesforce/cli-plugins-testkit';
+import { Duration } from '@salesforce/kit';
+import { asDictionary, AnyJson, get, getString, isArray } from '@salesforce/ts-types';
+
+const concatString = (a: string, b: string): string => a + b;
+
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+
+describe('Org Command NUT', function async() {
```

Comment:
could we make this an arrow function for consistency?

Reply:
Definitely not here because Pete's using the mocha `this` which the arrow loses.  
```
this.timeout(Duration.minutes(10).milliseconds);
```

Mocha's opinion on the subject: https://mochajs.org/#arrow-functions

Reply:
@WillieRuemmele, I am increasing timeout here and I need access to mocha's `this`. The difficulty I ran into was testing against local build. The setup takes longer, so overriding default applied in test:nuts.

I can remove the timeout and switch to arrow function.